src/Features/Onboarding.php

```diff
@@ -648,15 +647,28 @@ public static function delete_themes_transient() {
 	 * Add profiler items to component settings.
 	 *
 	 * @param array $settings Component settings.
+	 *
+	 * @return array
 	 */
 	public function component_settings( $settings ) {
 		$profile                = (array) get_option( self::PROFILE_DATA_OPTION, array() );
 		$settings['onboarding'] = array(
 			'profile' => $profile,
 		);
 
-		// Only fetch if the onboarding wizard OR the task list is incomplete or currently shown.
-		if ( ! self::should_show_profiler() && ! self::should_show_tasks() ) {
+		$is_wc_admin_page = false;
```

Comment:
Can you extract this out into a method like `should_show_profiler()` and `should_show_tasks()`? That would simplify this method a bit.

Reply:
@becdetat What do you think about extracting it out to a new method like `is_woocommerce_page()`. I checked both `should_show_profiler()` and `should_show_tasks()`, but I think the logic does not fit into the methods.

Reply:
@moon0326 oh yes, extracting it into a new method was what I meant :-) I could have explained that better

Reply:
@becdetat Oh haha now I read it again, it makes sense. Sorry about that 🤦

Reply:
@becdetat Updated in https://github.com/woocommerce/woocommerce-admin/commit/dfb2c2568ffac9078600c865bc88f8bfcb8b28c0